### DISCUSSION TITLE
【求教】 PAG.PAGFile.load(file)，返回了一个对象，报错的问题

### DISCUSSION THREAD

**Discussion by hubeishuaige** [Tencent/libpag#1979] [General]
上传一下代码：
![image](https://github.com/Tencent/libpag/assets/21007762/018d4a0b-0f10-42b4-93b0-0211a29621ea)

错误tis的图片：
![image](https://github.com/Tencent/libpag/assets/21007762/78af28e6-7559-44c8-a2a8-d7170df03cc4)

我就是想问一下，怎么把PAG.PAGFile.load(file)返回的数据挂到vue对象下面，方便在其它地方调用

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
